### DISCUSSION TITLE
Whenever I execute a command on the server PvPLegacy, I get an error. Can someone please help me out?

### DISCUSSION THREAD

**Discussion by TechPro424** [FabricMC/fabric-api#2809] [Player support] [answered]
Whenever I execute a command on the server PvPLegacy, I get an error. Can someone please help me out?
here is my modlist: https://moddermore.net/list/48528d77f8

**Comment by warjort** [comment]
You need to show the error. 

e.g. the logs/latest.log
Assuming whatever is breaking it is logging the error.

**Reply by TechPro424** [reply]
[latest.log](https://github.com/FabricMC/fabric/files/10328646/latest.log)

![javaw 2023-01-01 16-32-35](https://user-images.githubusercontent.com/86620085/210168437-2ec50c82-f330-4877-be51-b7e77ca4764d.png)

Here's the log and the error
I joined the server, executed the `/ping` command, got the error, and left

**Reply by warjort** [reply]
I think that means you are not authorised to the command and it is not reporting the error properly?

It's hard to see what is going in on in that log, it is full of opengl errors?

**Comment by apple502j** [comment]
This is a Fabric API bug. We have received countless reports of it, but nobody got it to succeed. I suspect this is caused by the server breaking some assumptions on networking stack.

@TechPro424 Is this a public server? I would like to test myself. If so please let me know how to join.

**Reply by TechPro424** [reply]
The IP of the server is `play.pvplegacy.net`

**Comment by kevinthegreat1** [comment]
Seems to be most likely caused by [this mod](https://www.curseforge.com/minecraft/mc-mods/xp-obelisk-additions). This is fixed in the newest version of this mod. Should be reproducible with this mod and `ecologics`.
https://github.com/Meridanus/fabric_xp_storage_additions/issues/18
https://discord.com/channels/507304429255393322/721100785936760876/1057414537424408616

**Reply by TechPro424** [reply]
But I don't have this mod installed on my client
And the server I got the error on does not have this mod either

**Comment by TechPro424** [comment, marked as answer]
Turns out, it was [this mod](https://modrinth.com/mod/wmch) that was causing the issue for me
The mod dev released an update today to fix [the issue](https://github.com/mrbuilder1961/WheresMyChatHistory/issues/49) and it worked
